### PR TITLE
fix: set default training mode to mixed_precision

### PR DESCRIPTION
**Pull request by williamstar**
expose the interface when  implement float32 training

### CODE REVIEW

File: persia/ctx.py [hunk 1]
```diff
@@ -415,7 +415,6 @@ def __init__(
         sparse_optimizer: Optimizer,
         dense_optimizer: torch.optim.Optimizer,
         device_id: int = 0,
-        mixed_precision: bool = False,
```
Comment (NOBLES5E) [inline persia/ctx.py:418]: I think this should be called fp16_embedding?
Mixed precision is already named for https://pytorch.org/docs/stable/amp.html which we also want to support later

**Review by NOBLES5E — CHANGES_REQUESTED** [review]
see comments

File: persia/ctx.py (hunk 1 above)
Comment (williamstar) [reply persia/ctx.py:418]: The default setting is `fp16_embedding`.  `fp16_embedding` argument will be add on when implement `f32_embeding` later on.
